corehq/apps/app_manager/static/app_manager/js/releases/releases.js

```diff
@@ -249,6 +249,7 @@ hqDefine('app_manager/js/releases/releases', function () {
         self.buildComment = ko.observable();
         self.upstreamBriefsById = _.indexBy(self.options.upstreamBriefs, '_id');
         self.upstreamUrl = self.options.upstreamUrl;
+        self.releasesEnabled = ko.observable(true);
```

Comment:
apologies for stressing on naming again.
can we think of better name for `releasesEnabled`, like `allowReleaseOperations`/`showReleaseOperations`?

Reply:
I always thought `showReleaseControl` was to show or hide the lock buttons but its enabling/disabling the release buttons.

so, `showReleaseControl` and `releasesEnabled` are doing the same thing right? just that one is a function and other is an attribute that we are setting.
I think then this could be the same name for both?